package.json

```diff
@@ -55,7 +55,7 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "@types/jest": "^24.0.0",
-    "@types/node": "^12.0.0",
+    "@types/node": "^12.20.55",
```

Comment:
If we're using Node 18, does this need to be bumped?

Reply:
I did update all modules, lets see if deployment works